### PR TITLE
web-ui: Deprecate Web UI in favor of openshift/console

### PR DESCRIPTION
**Pull request by mareklibra**
The Web UI project is deprecated, the Web Console [1] is
extended for virtualization support.

[1] https://github.com/openshift/console

### CODE REVIEW

File: administration/intro.adoc [hunk 1]
```diff
@@ -4,6 +4,8 @@ Installation
 KubeVirt is a virtualization add-on to Kubernetes and this guide assumes
 that a Kubernetes cluster is already installed.
 
+If installed on OpenShift, the web console is extended for management of virtual machines.
```
Comment (jelkosz) [inline administration/intro.adoc]: Should not we use the term OKD instead of openshift?

Comment (mareklibra) [reply administration/intro.adoc]: Good point. If so, other occurrences need to be updated as well.

**Comment by rawagner** [comment]
@fabiand please take a look

**Comment by iranzo** [comment]
@mareklibra can you please refresh the PR with signed commits? It will make the DCO test to pass as it is now a requirement. Thanks

**Comment by mareklibra** [comment]
@iranzo, sure, done

**Comment by mareklibra** [comment]
@fabiand , @iranzo , can you please have a look?

**Comment by iranzo** [comment]
Merging manually because the failing tests are just the rendering ones for the website

@fabiand ACK?

**Comment by fabiand** [comment]
Ack

**Pull request merged**